from app.api.v1.exceptions import GiftAppError


class PastEventError(GiftAppError):
    pass